website/docs/cli/commands/list/list-stacks.mdx

```diff
@@ -44,3 +44,59 @@ atmos list stacks -c vpc
 | Flag             | Description                                                                                                                                                                                                                               | Alias | Required |
 |------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------|----------|
 | `--component`    | Atmos component                                                                                                                                                                                                                         | `-c`  | no       |
+
+## Display Format
+
+The command output can be customized using template variables in your `atmos.yaml` configuration. Here's an example of how to configure the display format to adds custom fields called "Stack", "Tenant", "Environment", and "File:
```

Comment:
I don't think we describe anywhere what happens when using redirection or pipes, or that it automatically degrades when there is not tty attached